### DISCUSSION TITLE
vercel部署完成，但是无法使用

### DISCUSSION THREAD

**Discussion by Scheptical** [ChatGPTNextWeb/NextChat#1422] [Q&A]
已经按照教程设置好环境变量，但在使用过程中报错。
聊天窗口：
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/60955628/3314bfdd-e35d-4797-8666-199d8b13110d)

报错信息：
![image](https://github.com/Yidadaa/ChatGPT-Next-Web/assets/60955628/3608021b-412d-4b0c-8b75-d13a77afe7f6)

**Comment by vtea** [comment]
你把英文翻译下就知道原因了

**Reply by lxyntz** [reply]
openai key 的问题

**Reply by Scheptical** [reply]
我现在用zeabur部署成功了，可以正常使用，key也没问题，不知道是什么原因😂
